src/Tribe/Template.php

```diff
@@ -73,6 +73,15 @@ class Tribe__Template {
 	 */
 	protected $template_folder_lookup = false;
 
+	/**
+	 * Create a class variable for the include path, to avoid conflicting with extract.
+	 *
+	 * @since  4.11.0
+	 *
+	 * @var  string
+	 */
+	protected $template_current_file_path;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 05da6a73a2e8d8be81f0e8981e7aa54b</sub>